sdk/proto/go/provider.pb.go

```diff
@@ -894,7 +893,7 @@ func init() { proto.RegisterFile("provider.proto", fileDescriptor5) }
 
 var fileDescriptor5 = []byte{
 	// 840 bytes of a gzipped FileDescriptorProto
-	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x09, 0x6e, 0x88, 0x02, 0xff, 0xc4, 0x56, 0xdd, 0x6e, 0xc3, 0x34,
+	0x1f, 0x8b, 0x08, 0x00, 0x00, 0x00, 0x00, 0x00, 0x02, 0xff, 0xc4, 0x56, 0xdd, 0x6e, 0xc3, 0x34,
```

Comment:
any idea why this representation is changing?

Reply:
No idea. I'll check it out.

Reply:
So despite the digest changing, these two gzipped payloads (the old and the new) actually extract to identical contents.

Reply:
So THIS is why Bazel downloads its own Go toolchain...